app/javascript/admin/controllers/config_controller.js

```diff
@@ -93,6 +93,40 @@ export default class ConfigController extends Controller {
     }
   }
 
+  displaySnackbar(message) {
+    return document.dispatchEvent(
+      new CustomEvent('snackbar:add', { detail: { message } }),
+    );
+  }
+
+  async activateLightConfirmation(event) {
+    event.preventDefault();
+    try {
+      const data = new FormData(event.target);
```

Comment:
nitpick (non-blocking): You can call this `body` and use shorthand notation below, e.g.

```javascript
...
body,
...
```

instead of

```javascript
...
body: data,
...
```